README.md

```diff
@@ -95,7 +95,7 @@ CIP Editors meetings are public, recorded, and [published on Youtube](https://ww
 | 1694 | [A proposal for entering the Voltaire phase](./CIP-1694) | Proposed |
```

Comment:
Should we move up `1694` to below `1852`?

Reply:
yes of course... though we all merge-committed it like that, I swear someone else force-pushed it that way first 😂